packages/generators/app/lib/utils/db-client-dependencies.js

```diff
@@ -3,7 +3,7 @@
 const sqlClientModule = {
   mysql: { mysql: '2.18.1' },
   postgres: { pg: '8.6.0' },
-  sqlite: { sqlite3: '5.0.2' },
+  sqlite: { '@vscode/sqlite3': '5.0.7' },
```

Comment:
It doesn't appear like the original package has gone unmaintained and they are not redirecting to the microsoft one.  What's the reasoning here ?

Reply:
> It doesn't appear like the original package has gone unmaintained and they are not redirecting to the microsoft one. What's the reasoning here ?

There are various security issues in the vulnerabilities and mapbox isn't responding, Knex themselves moved to using the vscode fork and if we don't use it, Knex will throw errors if the sqlite dialect is used without that package installed.

Reply:
Ok I see, I'm not 100% comfortable ditching a package that was provided by a team for so long without thinking about it a bit more. I would like to see if there are plans for them to maintain the package in the long term or if they are letting Microsoft take over. Is there an issue on their repo with a conversation around that topic ?

 Right now the knex upgrade seems to be break all the tests anyway. Once that's fixed I guess we will have to consider it more

Reply:
I think that time long past https://github.com/knex/knex/issues/4858

Honestly if we want to go a better route we can drop both of these and move to better-sqlite3 which is a full fork with more features and has a proven history.